src/components/about-page/SoftwareExistsTo.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+import { useBookCallModal } from "../book-call/hooks/useBookCallModal";
+import "./css/software-exists-to.css";
+
+export const SoftwareExistsTo = () => {
+  const { modal, clickHandler } = useBookCallModal();
+
+  return (
+    <section className="software-exists-to">
```

Comment:
I would probably name this class `why-software-exists` - a noun or a complete thought would be a good rule for class naming